### DISCUSSION TITLE
FEATURE REQUEST: GAMING TWEAKS

### DISCUSSION THREAD

**Discussion by efyandro** [ChrisTitusTech/winutil#437] [Ideas]
It would be super cool if this tool can do a gaming tweak such as CPU Optimization, RAM Optimization, Latency Tweak, Network Optimization, or maybe Win32 Priority Separation. 
Adding MemReduct, Park Control and Process Lasso in application list

![firefox_Gi1UOlMefd](https://user-images.githubusercontent.com/5464033/204117621-b65100c7-9ffd-4b38-9e59-d4a480eeb115.jpg)

**Comment by nathsabari97** [comment]
Windows takes care of most these things and its pretty good at it. Mem reduct will do more harm than good.

**Reply by Sean-Wu-TW** [reply]
Interesting I thought I was mentioned 🤔

**Comment by anonyao** [comment]
@ChrisTitusTech  you should get msi installer for gpu/network/sata / chipset driver. But this scripts was nice. I have used your script always first thing i doo before connect internet. Or when i do it . And a quest ? Do u not uninstall MSstore apps.
